### DISCUSSION TITLE
Automatic Dark Taskbar and Taskbar Color Customization

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2824] [Ideas]
Automatic dark taskbar is to get the taskbar to automatically be dark in the evening, for example in the morning the taskbar is white, and in perhaps 5PM it's dark.

What I mean by color customization is to change your taskbar color to anything you want, Windows 11's color customizations are very minimal, if you want to make your taskbar a solid black color, it will say it is not supported, but in Windows 10 the taskbar could be a solid black color if you turn off transparency effects, in Windows 11, if you turn off the transparency effects, it doesn't really affect the way your taskbar is looking, it'll be a solid gray color rather than a solid black color.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Reply by TimMarvelHero** [reply]
posting the windows version and build isn't private info right?

**Reply by pyrates999** [reply]
No it is not.  Just post the version and build number.  You don't have to post the entire screen shot of winver.

**Reply by TimMarvelHero** [reply]
Version 23H2 (OS Build 22631.3155)

**Reply by TimMarvelHero** [reply]
![image_2024-02-15_135417809](https://github.com/valinet/ExplorerPatcher/assets/152800265/59063e7c-9129-4ab6-8a77-591b586ddfbb)

**Reply by TimMarvelHero** [reply]
<img width="592" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/56848209-9bbf-4b71-9cfe-eb4407d6966a">

**Reply by TimMarvelHero** [reply]
<img width="594" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/e8c08cfa-b853-4cd8-a053-bbf6c86c018a">

**Reply by TimMarvelHero** [reply]
<img width="589" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/989d1cc0-5da7-4ca7-ae56-fe2ad14858d6">

**Reply by TimMarvelHero** [reply]
<img width="595" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/3a9be734-79d3-4972-aaca-b6e75c1a11f9">

**Reply by TimMarvelHero** [reply]
<img width="592" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/c57c29dc-0ca4-49df-b448-54387a56610b">

**Reply by TimMarvelHero** [reply]
<img width="591" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/4bb0a5e5-21e0-4833-a398-8105a8b47f15">

**Reply by TimMarvelHero** [reply]
<img width="588" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/059ec0e8-679e-487e-9f20-0abe81c0463f">

**Reply by TimMarvelHero** [reply]
<img width="592" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/02cd723f-289d-4539-8e83-91b5043c7e0c">

**Reply by TimMarvelHero** [reply]
the only software that I I'm using that changes something in the taskbar is TranslucentTB.

**Reply by TimMarvelHero** [reply]
<img width="593" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/f8940dee-8cc7-4549-840a-2085a8bff577">

**Reply by pyrates999** [reply]
Ok, settings seems to be ok and you're up to date on the current version of EP.

**Reply by TimMarvelHero** [reply]
that's great. Is that needed for the idea I gave? The automatic dark mode

**Reply by pyrates999** [reply]
As already suggested in the bottom post here, please use TranslucentTB for colour customization.

**Reply by TimMarvelHero** [reply]
Yes, but it doesn't automatically set white/dark mode. It's manual.

**Reply by pyrates999** [reply]
Such a feature request has been made for TranslucentTB: https://github.com/TranslucentTB/TranslucentTB/issues/520

You should add that you'd like it too.

**Reply by TimMarvelHero** [reply]
Okay, thanks.

**Comment by Amrsatrio** [comment]
For color customization, please use TranslucentTB for this.
